src/test/java/core/basesyntax/files/FileReaderImplTest.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.files;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardOpenOption;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class FileReaderImplTest {
+    private static final String fileName = "temp.csv";
+    private static final Path path = Paths.get(fileName);
+    private static final FileReader fileReader = new FileReaderImpl();
+
+    @Test
+    public void readFromFile_Ok() throws IOException {
+        Files.deleteIfExists(path);
```

Comment:
you use this line in all test methods, move it to the method with annotation @AfterEach

Reply:
:)